ui/pages/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -177,6 +192,10 @@ const mapStateToProps = (state, ownProps) => {
 
   const methodData = getKnownMethodData(state, data) || {};
 
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  const accountType = getAccountType(state, fromAddress);
```

Comment:
why not place this line, in the code fence bellow, L216, and this way it's one less code fence in the file.